larreco/QuadVtx/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
+art_make(BASENAME_ONLY
+         LIB_LIBRARIES
+           larcorealg_Geometry
+           ${ART_FRAMEWORK_SERVICES_REGISTRY}
+           ${ROOT_CORE}
+           ${ROOT_HIST}
+           ${ROOT_PHYSICS}
+           canvas
+           cetlib_except
+         MODULE_LIBRARIES
+           larcorealg_Geometry
+           lardataobj_RecoBase
+           larreco_QuadVtx
+           nusimdata_SimulationBase
+           lardata_ArtDataHelper
+           ${ART_ROOT_IO_TFILESERVICE_SERVICE}
+           ${ART_ROOT_IO_TFILEDIRECTORY}
```

Comment:
There is no library called `${ART_ROOT_IO_TFILEDIRECTORY}`.  Please remove.

Reply:
Done